### ISSUE TITLE
[Bug]: Image cannot be delected by Del/Backspace key in notes editor

### ISSUE THREAD

**Issue by tomtomme** [jasp-stats/jasp-issues#2776]
tested with 0.18.3 and 0.19beta

### Description
I cannot delete images via select+del. I cannot add images via drag and drop.

### Purpose
I can delete an image via adding the cursor in front of it and hit backspace, but "del" is the more widely used key to delete something.

**Comment by shun2wang** [comment]
> select+entf

Do you mean cursor+<kbd>Del</kbd> right?

**Comment by tomtomme** [comment]
ah yes, german "entf" english "del"
